backend/exchanges/btcdirect.go

```diff
@@ -31,14 +36,31 @@ func GetBtcDirectSupportedRegions() []string {
 	return regions
 }
 
-// IsBtcDirectSupported is true if coin.Code and region are supported by BtcDirect.
-func IsBtcDirectSupported(coinCode coin.Code, region string) bool {
+// IsBtcDirectSupported is true if coin.Code is supported by BtcDirect.
+func IsBtcDirectSupported(coinCode coin.Code) bool {
 	supportedCoins := []coin.Code{
 		coin.CodeBTC, coin.CodeTBTC, coin.CodeLTC, coin.CodeTLTC, coin.CodeETH, coin.CodeSEPETH,
 		"eth-erc20-usdt", "eth-erc20-usdc", "eth-erc20-link"}
 
 	coinSupported := slices.Contains(supportedCoins, coinCode)
-	regionSupported := len(region) == 0 || slices.Contains(regions, region)
 
-	return coinSupported && regionSupported
+	return coinSupported
+}
+
+// BtcDirectDeals returns the purchase conditions (fee and payment methods) offered by BTCDirect.
+func BtcDirectDeals() *ExchangeDealsList {
+	return &ExchangeDealsList{
+		ExchangeName: BTCDirectName,
+		Deals: []*ExchangeDeal{
+			{
+				Fee:     4.9,
+				Payment: CardPayment,
+				IsFast:  true,
+			},
+			{
+				Fee:     1.9,
+				Payment: BankTransferPayment,
```

Comment:
I took these values from the info box that compare after clicking "next", but I'm not sure if they are 100% correct

Reply:
I know read it more carefully and those are placeholders values copied from the MoonPay one :D

Reply:
yeah they are just a copy, afaik we use those value in the in app guide

<img width="679" alt="Screenshot 2025-01-21 at 18 32 41" src="https://github.com/user-attachments/assets/3d31bc97-2f47-404b-85ec-8f043fdcd1ec" />

We don't need fee for the BTC Direct widget but I guess for the BTC Direct in-app guide.

Reply:
We need fees and payment methods to correctly load the badges in the `BuySell` compontent